lib/utils.js

```diff
@@ -36,7 +36,9 @@ export function grammarToLanguage(grammar: ?atom$Grammar) {
     l => l.toLowerCase() === grammarLanguage
   );
 
-  return kernelLanguage ? kernelLanguage.toLowerCase() : grammarLanguage;
```

Comment:
The way I understood `languageMappings` is that if it was set to `{ "babel": "javascript"}`, then a kernel that declares it can run `babel`, will be regarded by Hydrogen as a kernel that can run `javascript`.